src/Components/CameraFeed/CameraFeed.tsx

```diff
@@ -183,15 +187,18 @@ export default function CameraFeed(props: Props) {
             {props.children}
           </div>
           <div className="flex w-full flex-col items-end justify-end md:flex-row md:items-center md:gap-4">
-            <div className="flex items-center gap-2">
-              <span className="text-xs font-bold md:text-sm">
-                {props.asset.name}
-              </span>
-              <MonitorAssetPopover
-                asset={props.asset}
-                className="absolute z-[100] mt-2 w-56 -translate-x-full -translate-y-4 rounded-md bg-white md:w-[350px] md:-translate-x-full md:-translate-y-2"
-              />
-            </div>
+            {!hideMonitorAsset && (
+              <div className="flex items-center gap-2">
+                <span className="text-xs font-bold md:text-sm">
+                  {props.asset.name}
+                </span>
+                <MonitorAssetPopover
```

Comment:
Feel free to rename the `MonitorAssetPopover` component to `AssetInfoPopover`. It was initially built for HL7 Vitals Monitor and Ventilator Monitor Assets, but might have not been renamed make it more generic once we started using it for Camera Assets too.

```suggestion
                <AssetInfoPopover
```